frontend/packages/modelServing/src/concepts/ModelServingContext.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useResolvedExtensions } from '@odh-dashboard/plugin-core';
+import { ModelServingPlatform } from './modelServingPlatforms';
+import { isModelServingPlatformExtension } from '../extension-points';
+
+type ModelServingContextType = {
+  availablePlatforms?: ModelServingPlatform[];
+};
+
+export const ModelServingContext = React.createContext<ModelServingContextType>({
+  availablePlatforms: undefined,
+});
+
+type ModelServingProviderProps = {
+  children: React.ReactNode;
+};
+
+export const ModelServingProvider: React.FC<ModelServingProviderProps> = ({ children }) => {
+  const [platforms] = useResolvedExtensions(isModelServingPlatformExtension);
+
+  const contextValue = React.useMemo(
+    () => ({ availablePlatforms: platforms } satisfies ModelServingContextType),
+    [platforms],
+  );
```

Comment:
```
function useMemo<T>(factory: () => T, deps: DependencyList): T;
```
FWIW, Memo is a generic -- you can just provide the type to it instead of satisfies.